docs-source/content/quickstart/install.md

```diff
@@ -37,40 +52,77 @@ $ helm install stable/traefik \
   --wait
 ```
 
+
+For helm version 3.x, you should have added the stable repository if it has not been done already.
```

Comment:
@rosemarymarano I am not sure about this use of past tense- I don't think we use past tense anywhere else in the doc....

Reply:
I created a new section  Setup Helm now and let me know how to improve the formatting

Reply:
I have tested on 2.14.3 and 3.02.  I am surprised it also work in old 2.8.2

Reply:
@markxnelson True. This sentence has been removed :)